graph-builder/src/config/mod.rs

```diff
@@ -12,3 +12,6 @@ mod options;
 mod settings;
 
 pub use self::settings::AppSettings;
+
+/// Common prefix for graph-builder metrics.
+pub const METRICS_PREFIX: Option<&str> = Some("cincinnati_gb");
```

Comment:
Not sure if we want Option and Some here, but its okay to stay

Reply:
Agreed. Let me see how much effort it is to change that.

Reply:
Not much effort at all. PTAL